form/form_collections.rst

```diff
@@ -510,7 +510,7 @@ you will learn about next!).
 
         .. code-block:: yaml
 
-            # src/Resources/config/doctrine/Task.orm.yml
+            # src/Resources/config/doctrine/Task.orm.yaml
```

Comment:
I'm not sure if `yaml` ext is already supported for Doctrine :/ https://github.com/doctrine/doctrine2/blob/master/lib/Doctrine/ORM/Mapping/Driver/SimplifiedYamlDriver.php#L33

Reply:
That's sad. Reverted.